projects/legacy/library/ansible_toolbox.py

```
@@ -0,0 +1,369 @@
+"""
+ANSIBLE_OS_CONFIGURATIONS = yaml.safe_load(ANSIBLE_OS_CONFIGS_YAML)
+
+
+class Toolbox:
+    """
+    The Topsail Toolbox
+    """
+
+    def __init__(self):
+        for toolbox_file in (TOPSAIL_DIR / "projects").glob("*/toolbox/*.py"):
+            if toolbox_file.name.startswith("."): continue
```

Ruff (0.15.6)

[error] Multiple statements on one line (colon)

(E701)